src/encoded/schemas/file.json

```diff
@@ -203,13 +203,16 @@
     },
     "facets": {
         "file_type" : {
-            "title" : "File Type"
+            "title" : "File Type",
+            "descripton" : "Type or categorization of this file."
         },
         "file_format": {
-            "title" : "File Format"
+            "title" : "File Format",
+            "description" : "Format of the file, i.e. the file extension."
         },
         "experiments.experiment_type" : {
-            "title" : "Experiment Type"
+            "title" : "Experiment Type",
+            "description" : "Type of experiment of which this file is a part of."
```

Comment:
I know it's just semantics but might look/sound a little better as "Type of experiment to which this file belongs"